app/models/manageiq/providers/vmware/infra_manager/vm/reconfigure.rb

```diff
@@ -262,6 +269,29 @@ def remove_disk_config_spec(vim_obj, vmcs, hardware, options)
     end
   end
 
+  def resize_disk_config_spec(vim_obj, vmcs, hardware, options)
+    raise "resize_disk_config_spec: disk filename is required." unless options[:disk_name]
+
+    device = vim_obj.getDeviceByBacking(options[:disk_name], hardware)
+    raise "resize_disk_config_spec: no virtual device associated with: #{options[:disk_name]}" unless device
+    raise "resize_disk_config_spec: decrease size is not supported for: #{options[:disk_name]}" unless device.capacityInKB.to_i <= options[:disk_size_in_mb].to_i * 1024
+
+    add_device_config_spec(vmcs, VirtualDeviceConfigSpecOperation::Edit) do |vdcs|
+      vdcs.device = VimHash.new("VirtualDisk") do |dev|
+        dev.key           = device.key
+        dev.capacityInKB  = options[:disk_size_in_mb].to_i * 1024
```

Comment:
Since you're using `options[:disk_size_in_mb].to_i * 1024` twice can you just save it as a local var e.g. `new_capacity_in_kb`?

Reply:
✔️ I will improve this